docs/cloud.md

```diff
@@ -0,0 +1,234 @@
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/): to interact with the deployed EKS cluster.
+* [Helm](https://helm.sh/docs/intro/quickstart/): to deploy and upgrade Feldera Cloud on EKS.
+* A Feldera license, as part of which you will receive a Helm Chart and container artifacts to run Feldera Cloud.
+
+## Deployment overview
+
+We will go through the following steps to deploy Feldera Cloud on EKS:
+
+* Use `eksctl` to create a dedicated VPC and bring up an EKS cluster inside it.
+* Use `Helm` to deploy Feldera Cloud inside the EKS cluster.
+* Configure input and output data sources on EKS.
+* Connect these data sources to Feldera.
+
+
+## Create an EKS cluster
+
+Make sure your `aws` CLI is configured to work with your AWS account. If you haven't done so already,
+run `aws configure` and enter the required credentials.
+
+Next, create a configuration file named `eks-config.yaml` for running `eksctl`.
+
+
+```
+# File name: eks-config.yaml
+
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+metadata:
+  name: feldera-eks
+  region: us-west-1
+  version: "1.27"
+
+# OIDC provider is needed to use EBS
+iam:
+  withOIDC: true
+
+managedNodeGroups:
+  - name: managed-ng
+    desiredCapacity: 3
+    instanceType: m5.large
+    privateNetworking: true
+
+vpc:
+  clusterEndpoints:
+    privateAccess: true
+    publicAccess:  true
+    # publicAccessCIDRs: ["<a.b.c.d/32>"] # Change to your IP address if using publicAccess: true
+
+addons:
+- name: aws-ebs-csi-driver
+  wellKnownPolicies:
+    ebsCSIController: true
+```
+
+To create an EKS cluster with this configuration, run:
+
+```
+eksctl create cluster -f eks-config.yaml
+```
+
+This uses CloudFormation behind-the-scenes to bring up an EKS cluster named
+`feldera-eks` with three worker nodes (m5.large EC2 instances, specified in
+managedNodeGroups), running in a newly created dedicated VPC. It should take
+roughly 15-20 minutes.
+
+`eksctl` will create the VPC in the `us-west-1` region which will run the EKS
+cluster. The cluster will be deployed across two availability zones (AZs). Each
+AZ will have two subnets, one private and one public. The private subnets will
+have outbound Internet connectivity via a NAT gateway. The public subnets will
+have inbound and outbound connectivity via an Internet Gateway. The worker
+nodes will only be on the private subnet.
+
+
+:::tip
+Unless you are running `eksctl` from a machine which is in the same VPC and
+subnet as the EKS control plane, you will want to set `publicAccess: true`.
+If so, we strongly recommend using the `publicAccessCIDRs` setting to limit IP ranges
+which are allowed to interact with the EKS control plane. For example, set it
+to the publicly visible IP of the machine where you are running these commands.
+
+See the [eksctl cluster access](https://eksctl.io/usage/vpc-cluster-access/)
+documentation to learn more.
+:::
+
+:::tip
+`eksctl` allows extensive configuration beyond what we've shown here. Check out
```

Comment:
> allows extensive configuration beyond what we've shown here.

allows additional Cluster Access configuration changes beyond the basic settings above.